packages/ts/react-grid/src/header-filter.tsx

```diff
@@ -5,9 +5,24 @@ import { Select, type SelectElement } from '@hilla/react-components/Select.js';
 import { TextField, type TextFieldElement } from '@hilla/react-components/TextField.js';
 import { type ReactElement, type RefObject, useContext, useEffect, useRef, useState } from 'react';
 import { ColumnContext } from './autogrid-column-context.js';
-import css from './header-filter.module.css';
 import Matcher from './types/dev/hilla/crud/filter/PropertyStringFilter/Matcher';
 
+const autoGridFilterWithLessGreaterEqualsStyle = document.createElement('style');
```

Comment:
Let's have a `TODO` mark here to remember that we would like to replace it later.
```suggestion
// TODO: Replace with more robust solution
const autoGridFilterWithLessGreaterEqualsStyle = document.createElement('style');
```